docs/Contributing/Testing-and-local-development.md

```diff
@@ -486,71 +486,26 @@ To enable the [DEP](https://github.com/fleetdm/fleet/blob/main/tools/mdm/apple/g
 1. A certificate.
 1. An encrypted token generated by Apple.
 
-#### Private key + certificate
+#### Private key, certificate, and encrypted token
 
-You can generate the private key and the certificate using `fleetctl`:
+First ask @zwass to create an account for you in [ABM](https://github.com/fleetdm/fleet/blob/main/tools/mdm/apple/glossary-and-protocols.md#abm-apple-business-manager). You'll need an account to generate an encrypted token.
```

Comment:
Who's the audience for this? Our team only? If the audience for this is the wide community, then this doesn't seem workable.

Reply:
Hmm, I think the audience is for the wider community. Anyone can contribute so anyone should be able to follow instructions in the contributor docs.

That said, I think this is a separate problem. The "ask Zach" instructions already exist. 

My edits to the contributor docs in this PR is focused on depulicating the generate certs/keys instructions. These changes solve the problem of maintaining two places in the docs with the same instructions.